.github/workflows/ci-workflow.yml

```diff
@@ -93,6 +94,17 @@ jobs:
           ${{ env.MAVEN_COMMAND }}
           ${{ env.MAVEN_PROFILES }}
           ${{ env.MAVEN_PROJECTS }}
+      - name: upload-on-failure
+        uses: actions/upload-artifact@v2
+        with:
+          name: surefire-reports-ubuntu-en
+          path: |
+            ./**/target/surefire-reports/*.txt
+            ./**/target/surefire-reports/*.xml
+          retention-days: 3
+        if: failure()
+      - name: System Information (after)
```

Comment:
It is probably worth renaming this to Disk Usage or something similar.

Reply:
I chose this for symmetry with the preceding "System Information" label, which collects data before the build.  My thought was that there might be other non-disk information we might also want to collect after the build.  But open to a different name.